### PR TITLE
Fixed path to certfile, added debug echos

### PR DESCRIPTION
**Pull request by SilverFire**
Follow-up to #379

@buchdag in the previous pull request the `certfile` variable did not contain the full path to cert, so the patch was pretty useless. This pull request is checked better and fixes my mistake.

### CODE REVIEW

File: app/letsencrypt_service [hunk 1]
```diff
@@ -86,22 +86,29 @@ function cleanup_links {
...
           # If certificate is not letsencrypt, don't ever try to remove it
           if [[ -f "${certfile}" ]] && [[ -z $(openssl x509 -noout -issuer -in ${certfile} | grep "Let's Encrypt") ]]; then
+              echo "certificate is not LE. Skipping."
               continue
-          fi;
+          fi
           for extension in .crt .key .dhparam.pem .chain.pem; do
               file="${disabled_domain}${extension}"
+              echo -n -e "\nChecking ${file}"
               if [[ -n "${file// }" ]] && [[ -L "/etc/nginx/certs/${file}" ]]; then
```
Comment (SilverFire) [inline app/letsencrypt_service]: I wonder why `[[ -n "${file// }" ]]` works as expected. Could you give me a hint how to explain it?

Comment (SilverFire) [reply app/letsencrypt_service]: Wow, thank you for the detailed explanation. I rarely code something on bash, but always discover something interesting in syntax.

P.S.  First time meet developer from Nantes on GitHub :) I has a talk last year in Nantes on PHP conference PHP Tour. You have a beautiful city and wonderful Elephant)

**Comment by buchdag** [comment]
The debug output is still a bit too verbose to my taste but I'll go ahead and merge it like this.

File: app/letsencrypt_service (hunk 1 above)
Comment (buchdag) [reply app/letsencrypt_service]: He he thank you, I'm relatively new in Nantes but yes that's one hell of a nice city 😃

**Pull request merged**